adserver/analyzer/migrations/0005_remove_embedding.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 4.2.11 on 2024-03-14 18:53
```

Comment:
This migration should not be run until the deploy is out already. Normally I just "deprecate" fields to be cleaned up later.